src/legacy/core_plugins/kibana/public/discover/np_ready/angular/discover.js

```diff
@@ -943,15 +977,6 @@ function discoverController(
     $scope.minimumVisibleRows = $scope.hits;
   };
 
-  $scope.updateSavedQueryId = newSavedQueryId => {
-    if (newSavedQueryId) {
-      $state.savedQuery = newSavedQueryId;
-    } else {
-      delete $state.savedQuery;
-    }
-    $state.save();
-  };
-
   async function setupVisualization() {
     // If no timefield has been specified we don't create a histogram of messages
     if (!$scope.opts.timefield) return;
```

Comment:
If you're removing this variable, lets remove it everywhere?

Reply:
you mean $scope.opts.timefield? didn't intend to since it's also used in discover.html template

Reply:
I just saw you replaced it in a couple of places with getTimeField

Reply:
note: I do agree it should go away, and it will with further deangularization